docs/source/tutorial/starting.rst

```diff
@@ -56,6 +56,44 @@ install with the command ``make install``.  This will, by default, install into
 
     IDRIS2_PREFIX ?= /usr/local
 
+On Windows
+-------------------------
+
+On Windows systems you can build and use Idris 2 from sources using WSL (Windows Subsystem for Linux).
+
+Open Windows PowerShell as administrator and run:
+
+.. code-block:: PowerShell
+
+	wsl --instal # enable Window to run a linux system 
+	wsl --install -d Ubuntu # install a specific linux distro
+
+A restart may be required.
+
+Open PowerShell and run:
+
+.. code-block:: PowerShell
+
+	wsl -d Ubuntu # open a linux terminal
+	apt update # it is necessary to install dependencies
+	apt install make gcc chezscheme libgmp3-dev # install dependencies required for build from source
```

Comment:
* You need to use `sudo` for package installations.
* There is a `build-essential` meta-package which installs all the relevant tools (`make`, `gcc`, etc.)
  - Chez and GMP will still need a separate listing to be installed.
* Probably good to ask them to install the man-pages (documentation) as well: `manpages-dev` ?

Reply:
default wls installation has root user only (at least from what i know). But good idea to prepend sudo just in case the user created other linux users. Ok, i will update to build-essentials. What would the user need the man-pages for?
Thank you for all the precious feedback! (I'm learning a lot)

Reply:
The [WSL documentation](https://learn.microsoft.com/en-us/windows/wsl/setup/environment#set-up-your-linux-username-and-password) seems to suggest otherwise:
> Once the process of installing your Linux distribution with WSL is complete, open the distribution (Ubuntu by default) using the Start menu. You will be asked to create a User Name and Password for your Linux distribution.

The man-pages would mostly be for trouble-shooting on the user's end. I guess they're not strictly required, but a lot of guides, StackOverflow answers, and similar refer to them as a reference point and/or further reading. So I figured it'd be best to include them  : )